### PR TITLE
Add a pipeline to deploy and test a release of a Tekton project

### PR DESCRIPTION
**Pull request by afrittoli**
Add a Task that gets a release of a Tekton project from the release
bucket, and installs it in a target cluster.

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Comment by afrittoli** [comment]
Added "hold" to allow for a couple of reviews. This is not used anywhere yet but I'm building a pipeline to test releases bit by bit.

**Comment by afrittoli** [comment]
Related to #75

File: scripts/logs/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# How to follow log outputs
```
Comment (vdemeester) [inline scripts/logs/README.md]: Shouldn't we just point to `tkn` instead ?

File: tekton/README [hunk 2]
```diff
@@ -0,0 +1,213 @@
+# Tekton Resources for CI/CD
```
Comment (vdemeester) [inline tekton/README]: The file should be `README.md` :stuck_out_tongue:

Comment (afrittoli) [reply tekton/README]: heh, nice catch, thanks

File: scripts/logs/README.md (hunk 1 above)
Comment (afrittoli) [reply scripts/logs/README.md]: this is copied straight from pipeline - we might do something different eventually - but I wanted to have a smooth migration, by using the existing scripts mostly wrapped into tasks

**Comment by afrittoli** [comment]
I think this is in a reasonable state to go into the repo now, and then iterate from here.

File: tekton/README (hunk 2 above)
Comment (afrittoli) [reply tekton/README]: Done

File: scripts/logs/README.md (hunk 1 above)
Comment (afrittoli) [reply scripts/logs/README.md]: I've seen your PR about this, once it's done on pipelines I can port it to plumbing if that's ok.

**Pull request merged**